common/djangoapps/entitlements/migrations/0007_remove_courseentitlementsupportdetail_reason.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('entitlements', '0006_courseentitlementsupportdetail_action'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='courseentitlementsupportdetail',
+            name='reason',
```

Comment:
`reason` is used in the [`EntitlementSupportView`](https://github.com/edx/edx-platform/blob/9cf7f4509912c4087227ef959206dea424b5beb2/lms/djangoapps/support/views/course_entitlements.py#L19) and the [`CourseEntitlementSupportDetailsSerializer`](https://github.com/edx/edx-platform/blob/9cf7f4509912c4087227ef959206dea424b5beb2/common/djangoapps/entitlements/api/v1/serializers.py#L34) - please excuse my Django ignorance, but will removing this field break the behavior in the View or Serializer?

Reply:
Yes it should definitely be removed from the serializer `fields` tuple. I have a PR up that will actually remove the `EntitlementSupportView` altogether though, because we're migrating the support actions to the existing entitlements endpoint.